.github/workflows/e2e-atomic-fruits-vault.yml

```diff
@@ -1,4 +1,4 @@
-name: Temporary job to test all steps to install kind, vault & crossplane
+name: Job testing and end to end scenario with simple application. It tests all steps from kind and vault installation.
```

Comment:
-> "Job testing an end to end scenario with a simple Quarkus application. It tests all the steps starting from kind and vault installation."